components/00-base/01-colors/_00-color-functions.scss

```diff
@@ -0,0 +1,199 @@
+// Convert color values to desire format.
+// Returns hex by default.
+// Optionally returns rgb, rgba, hsl, or hsla.
+@function cnvrt-clr($color, $format: null) {
+
+  @if (type-of($color) == color) {
+
+    @if ($format == null or $format == 'hex') {
+      $ie-hex: ie_hex_str($color);
+      @return unquote('#' + str_slice($ie-hex, 4));
+    }
+    @else if ($format == 'rgb') {
+      @return red($color), green($color), blue($color);
+    }
+    @else if ($format == 'rgba') {
+      @return red($color), green($color), blue($color), opacity($color);
+    }
+    @else if ($format == 'hsl') {
+      @return hue($color), saturation($color), lightness($color);
+    }
+    @else if ($format == 'hsla') {
+      @return hue($color), saturation($color), lightness($color), opacity($color);
+    }
+    @else {
+      @error "#{$format} is not a valid color format";
+    }
+  }
+  @else {
+    @error "#{$color} is not a valid color";
+  }
+}
+// This color function makes it easy for you to call a color by it's
+// functional name anywhere you need.
+// E.g. to use the `text` color, you type: `color: clr(text);`
+//
+// How it works:
+// This function works in conjunction with the dynamic css custom properties
+// defined in the _01-color-definitions.scss. The item called in the
+// function is prefixed with `var(--c-` to call the custom property for that color.
+// E.g. `color: clr(text);` will return `color: hsla(var(--c-text), 1);`
+// This function also accepts an alpha number argument that will
+// return the color value as (hsl or rgb) + alpha.
+@function clr($name, $variant:null, $alpha: 1, $theme:default, $format:hsl) {
```

Comment:
Default formatting is `hsl` so the function can output the full color value as `hsla`. Change this option to `rgb` so the function output is `rgba`.